artemis-server/src/main/java/org/apache/activemq/artemis/core/server/federation/FederatedQueueConsumer.java

```diff
@@ -163,11 +181,29 @@ private void disconnect() throws ActiveMQException {
    @Override
    public void onMessage(ClientMessage clientMessage) {
       try {
+         if (server.hasBrokerFederationPlugins()) {
+            try {
+               server.callBrokerFederationPlugins(plugin -> plugin.beforeFederatedQueueConsumerMessageHandled(this, clientMessage));
+            } catch (ActiveMQException t) {
+               logger.warn("Error executing beforeFederatedQueueConsumerMessageHandled plugin method: {}", t.getMessage(), t);
```

Comment:
should be a logger with code, e.g. explicit method logger with code as at warn (same for all logger.warns in this PR)